src/main/java/org/apache/commons/lang3/reflect/TypeUtils.java

```diff
@@ -1178,11 +1188,15 @@ public static boolean isInstance(final Object value, final Type type) {
      * @return a non-empty array containing the upper bounds of the wildcard
      * type.
      */
-    public static Type[] getImplicitUpperBounds(final WildcardType wildcardType) {
+    @SuppressWarnings("index:return.type.incompatible") /*
+    #1: bounds.length != 0 ensures @MinLen(1)
```

Comment:
It is also important, that `normalizeUpperBounds` returns non-empty result for a non-empty argument.